universal-login-commons/test/react/getTokens.test.ts

```diff
@@ -0,0 +1,9 @@
+import {expect} from 'chai';
+import {getErc721Tokens} from '../../src/react/getErc721Tokens';
+
+describe('UNIT: getTokens', () => {
+  it('Rinkeby: Get tokens of 0x7c6A49cD07EcB899025fFcF489561B5f499b07E1', async () => {
+    const tokens = await getErc721Tokens('0x7c6A49cD07EcB899025fFcF489561B5f499b07E1', 'rinkeby');
```

Comment:
We don't want to have such a test. We should mock these API responses in the test.